tests/generic_file_test.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from file_metadata.generic_file import GenericFile
+from tests.utilities import fetch_file
+
+
+class DerivedFile(GenericFile):
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tests/generic_file_test.py
+++ b/tests/generic_file_test.py
@@ -5,6 +5,7 @@
 
 
 class DerivedFile(GenericFile):
+
     def analyze(self):
         # Only use the `_analyze_test` functions for tests
         return GenericFile.analyze(self, prefix='_analyze_test')
```